app/view/partials/sign-up-form.ejs

```diff
@@ -1,18 +1,9 @@
-<form action="/" method="POST">
-    <label>Username
-        <input type="text" name="username">
-    </label>
-    <label>Password
-        <input type="password" name="password">
-    </label>
-    <label>First name
-        <input type="text" name="firstname">
-    </label>
-    <label>Last name
-        <input type="text" name="lastname">
-    </label>
-    <label>Your bike
-        <input type="text" name="bike">
-    </label>
-    <button type="submit">Sign up</button>
-</form>
\ No newline at end of file
+<button class="btn button--fullwidth" type="button" name="button" id="untappdConnect" onclick="">Connect with Untappd</button>
+<span class="form__or">or</span>
+<form class="sign-up__form" action="/" method="POST">
+    <label>Username</label>
+    <input type="text" name="username" placeholder="username">
```

Comment:
`<label>` Klik mij `<input type="text"></label>` Place the label around input fields so if the user clicks on it it will select the input fields